src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,29 @@
+package calculator;
+
+import java.util.Scanner;
+
+public class StringCalculator {
+
+	public static void main(String[] args) {
+		Scanner scanner = new Scanner(System.in);
+		String inputString = scanner.nextLine();
```

Comment:
```suggestion
		String input = scanner.nextLine();
```
타입을 보면 String인걸 알 수 있으니 변수명에 쓸 필요는 없습니다